webpack/assets/javascripts/bundle.js

```diff
@@ -6,6 +6,7 @@ require('jquery.cookie');
 require('./bundle_flot');
 require('./bundle_select2');
 require('./bundle_datatables');
-window.tfm = {
-  tools: require('./foreman_tools')
-};
+
+window.tfm = window.tfm || {};
```

Comment:
This is safer. If window.tfm already exists, any defined properties will be wiped out.